### ISSUE TITLE
JoinedText

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#205]
Let's create `JoinedText`, which will do what String.join() in doing:
```java
String joined = new JoinedText(" ", "hello", "world").asString();
```
And `joined` will be equal to `hello world`

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by fabriciofx** [comment]
Closed by #206.
